ports/daw-header-libraries/portfile.cmake

```diff
@@ -0,0 +1,23 @@
+set(VCPKG_BUILD_TYPE release)
+
+vcpkg_from_github(
+    OUT_SOURCE_PATH SOURCE_PATH
+    REPO beached/header_libraries
+    REF 760902a78b23ebc79c2ac51384d081df8bc8022f
+    SHA512 36d36d5cfe591b179ba9bc0ab3da0a035ddefa5c97a8b56e80c9b80e216fd36fb0e8c94a60d8861efdc2bbb29370c9bad796bd527e0d44afe0f8eaf5a9f2816a
```

Comment:
Just so I understands, this means that one has to update these prior to any push to release?

Reply:
Yes. If the semver version doesn't change, but the port does change, then the [port version](https://github.com/microsoft/vcpkg/blob/master/docs/users/versioning.md#port-version) has to be updated.